SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -287,6 +286,19 @@ bool FLocalDeploymentManager::LocalDeploymentPreRunChecks()
 		}
 	}
 
+	FSpotServiceInfoResult ServiceInfo = GetServiceInfo();
```

Comment:
Just realised. Is this adding any significant (greater than 0.5s) time delay to starting a local deployment? 
Since this calling an executable now it might slow down day to day development. We have an NFR for deployment launching taking no more than 5s in total from clicking start to the deployment being ready (we are already at 4.5s)
Could you please check how much time this is adding :D